### ISSUE TITLE
Prisma Cannot find package

### ISSUE THREAD

**Issue by lassejlv** [oven-sh/bun#6115]
### What version of Bun is running?

latest

### What platform is your computer?

Using railway.app

### What steps can reproduce the bug?

Getting this error while using a dockerfile for my project using prisma.

```bash
error: Cannot find module ".prisma/client/index" from "/usr/src/app/node_modules/@prisma/client/index.js"
```

here is my dockerfile

```docker
# Use the official Bun image
FROM oven/bun:latest

# Set the Docker working directory as /usr/src/app
WORKDIR /usr/src/app

# Copy everything from the current directory into Docker's /usr/src/app
COPY . .

RUN bun install

# Run the project
CMD bunx prisma generate && bun run src/index.js
```

### What is the expected behavior?

Install packages, and generate schema. Using Mongodb with prisma. And run my project

**Comment by Electroid** [comment]
Duplicate of #4959

**Comment by Electroid** [comment]
This is likely an issue where the prisma `postinstall` is not being run, check the linked issue for a temporary workaround.

**Comment by lassejlv** [comment]
@Electroid still getting the same issue

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway and @paperdave in #7132

This will be part of the Bun v1.0.17 release, which ships in a few hours from the time of writing.
